packages/core/src/internal/utils/supports.spec.ts

```diff
@@ -1,16 +1,11 @@
 /*
- * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import { supportsFlexGap, supportsResizeObserver } from './supports.js';
-
 describe('browser support feature checks', () => {
   it('should check if browser supports ResizeObserver', () => {
```

Comment:
Is the title of this test correct? The test assertion doesn't seem to answer it.

Reply:
nope, will fix

Reply:
fixed